macros/src/node.rs

```diff
@@ -2,42 +2,56 @@ use proc_macro2::{Ident as ProcIdent, Span};
 use quote::quote;
 use syn::Ident;
 
-// Generates the node wrapper to bind with JS
+// Generates a wrapper for the Audio Node, binding it to JavaScript
 pub fn node_wrapper(ident: &Ident) -> proc_macro2::TokenStream {
+    // Create a new identifier for the Node struct
     let node_ident = ProcIdent::new(&format!("{ident}Node"), Span::call_site());
 
+    // Generate the TokenStream for the wrapper
     quote! {
-      #[wasm_bindgen(js_name = #ident)]
-      pub struct #node_ident (waw::node::Node<#ident>);
-
-      #[wasm_bindgen(js_class = #ident)]
-      impl #node_ident {
-          pub async fn create(
-            ctx: waw::web_sys::AudioContext,
-            initial_state: Option<<#ident as waw::worklet::AudioModule>::InitialState>
-          ) -> Result<#node_ident, wasm_bindgen::JsValue> {
-              let result = waw::node::Node::<#ident>::create(ctx, initial_state).await?;
-              Ok(#node_ident(result))
-          }
-          pub fn node(&self) -> Result<waw::web_sys::AudioWorkletNode, wasm_bindgen::JsValue> {
-            Ok(self.0.inner.clone())
-          }
-
-          pub fn get_param(&self, param: <#ident as waw::worklet::AudioModule>::Param) -> waw::web_sys::AudioParam {
```

Comment:
in this code part, i think it is good if u add the comment for easy understanding